installer/engine_installer/engine_installer.go

```diff
@@ -14,8 +14,172 @@
 
 package main
 
-import "fmt"
+import (
+	"html/template"
+	"strings"
 
-func main() {
-	fmt.Println("hello world")
+	"context"
```

Comment:
Can you elaborate? I'm not sure what convention is and goimports was telling at me a lot.